### PR TITLE
Nbo/add run type to scenario manager prod 10350

### PR DESCRIPTION
**Pull request by nborde-CSM**

### CODE REVIEW

File: src/charts/ScenarioManagerTreeList/ScenarioManagerTreeList.js [hunk 1]
```diff
@@ -283,7 +284,8 @@ ScenarioManagerTreeList.propTypes = {
 ScenarioManagerTreeList.defaultProps = {
   onScenarioRedirect: null,
   labels: {
-    status: 'Run status',
+    status: 'Run status:',
+    runTemplate: 'Run type:',
```
Comment (csm-thu) [inline src/charts/ScenarioManagerTreeList/ScenarioManagerTreeList.js:288]: run type -> run template :slightly_smiling_face:

File: src/cards/ScenarioNode/ScenarioNode.js [hunk 2]
```diff
@@ -405,6 +412,7 @@ ScenarioNode.defaultProps = {
   showDeleteIcon: false,
   labels: {
     status: 'Run status:',
+    runTemplate: 'Run type:',
```
Comment (csm-thu) [inline src/cards/ScenarioNode/ScenarioNode.js:415]: "run type" -> "run template" :slightly_smiling_face:

File: src/cards/ScenarioNode/ScenarioNode.js [hunk 3]
```diff
@@ -368,6 +374,7 @@ ScenarioNode.propTypes = {
    */
   labels: PropTypes.shape({
     status: PropTypes.string.isRequired,
+    runTemplate: PropTypes.string.isRequired,
```
Comment (csm-thu) [inline src/cards/ScenarioNode/ScenarioNode.js:377]: Adding a required key in the labels dict is a breaking change

File: src/cards/ScenarioNode/ScenarioNode.js [hunk 4]
```diff
@@ -260,13 +263,15 @@ export const ScenarioNode = ({
   const getAccordionDetails = () => {
     return (
       <AccordionDetails className={classes.scenarioDetailsContainer}>
-        <div className={classes.scenarioDetailsNameLine}>{getScenarioNameAndValidationStatus(false)}</div>
+        <div className={classes.scenarioDetailsNameLine}>{getScenarioNameStatusAndRunType(false)}</div>
         {getDetailedStatus()}
+        <Typography className={classes.cardLabel}>{labels.runTemplate}</Typography>
+        <Typography data-cy="scenario-run-template" className={classes.runTemplate}>
```
Comment (csm-thu) [inline src/cards/ScenarioNode/ScenarioNode.js:269]: Can we rename `classes.runTemplate` to `classes.runTemplateName`? We're using `labels.runTemplate` just above, but it doesn't use this "class", so the name is misleading

File: src/cards/ScenarioNode/ScenarioNode.js [hunk 5]
```diff
@@ -189,10 +185,13 @@ export const ScenarioNode = ({
     );
   };
 
-  const getScenarioNameAndValidationStatus = (isValidationChipClickable = true) => {
+  const getScenarioNameStatusAndRunType = (isValidationChipClickable = true) => {
```
[the remark this answers is not in the document]
Comment (nborde-CSM) [reply src/cards/ScenarioNode/ScenarioNode.js:190]: There is already several fooDetails function or variable which can create confusion. What about getSenarioBasicInformations ?

File: src/cards/ScenarioNode/ScenarioNode.js (hunk 4 above)
Comment (nborde-CSM) [reply src/cards/ScenarioNode/ScenarioNode.js:269]: In the style file, the class _runTemplate_ is just above the datasets class which get the same attributes.

File: src/cards/ScenarioNode/ScenarioNode.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (csm-thu) [reply src/cards/ScenarioNode/ScenarioNode.js:190]: Discussed with NBO, we can try to refactor the code

File: src/cards/ScenarioNode/ScenarioNode.js [hunk 6]
```diff
@@ -368,6 +378,7 @@ ScenarioNode.propTypes = {
    */
   labels: PropTypes.shape({
     status: PropTypes.string.isRequired,
+    runTemplateLabel: PropTypes.string.isRequired,
```
Comment (csm-thu) [inline src/cards/ScenarioNode/ScenarioNode.js:381]: Adding a new required key in the labels dict is a breaking change :smile:

**Review by csm-thu — APPROVED** [review]
LGTM :+1:

**Pull request merged**